src/types.ts

```diff
@@ -89,3 +92,68 @@ export interface AmplitudeReactNativeModule {
     eventUploadThreshold: number,
   ): Promise<boolean>;
 }
+
+/**
+ * Strings that have special meaning when used as an event's type
+ * and have different specifications.
+ */
+export enum SpecialEventType {
+  IDENTIFY = '$identify',
+  GROUP_IDENTIFY = '$groupidentify',
+}
+
+export type BaseEvent = {
+  readonly event_type: Exclude<string, SpecialEventType>;
```

Comment:
Let's make these properties camelCase (eventType) to match the naming convention in this lib.

Reply:
fixed